src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="#Home">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
 
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a href="#Apple" class="nav__link">Apple</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Samsung</a>
+            <a href="#Samsung" class="nav__link">Samsung</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Smartphones</a>
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Laptops & computers</a>
+            <a href="#Laptops&Computers" class="nav__link">Laptops & computers</a>
```

Comment:
```suggestion
            <a href="#Laptops_Computers" class="nav__link">Laptops & computers</a>
```
Better avoid using special chars in hashes.